form/form_customization.rst

```diff
@@ -87,6 +87,43 @@ control over how each form field is rendered, so you can fully customize them:
     Later in this article you can find the full reference of these Twig
     functions with more usage examples.
 
+Form Field Helpers
+------------------
+
+The ``form_*()`` helpers render each part of the form field, including all its needed HTML elements. Most developers
+like this behavior, but some designers struggle with it, because it hides all the HTML in form themes which are not
+easy to manage by them.
+
+That's why some Twig form helpers are available to render the value of each form field part without adding any
+HTML around it:
+
+* ``field_name``
+* ``field_value``
+* ``field_label``
+* ``field_help``
+* ``field_errors``
+* ``field_choices`` (an iterator of the field choices; e.g. for ``<select>``)
+
+When using these helpers, you must write all the HTML contents for all form fields, which some people prefer to better
+control the generated HTML without having to deal with form themes:
+
+.. code-block:: html+twig
+
+    <input
+        name="{{ field_name(form.username) }}"
+        value="{{ field_value(form.username) }}"
+        placeholder="{{ field_label(form.username) }}"
+        class="form-control"
+    />
+
+    <select name="{{ field_name(form.country) }}" class="form-control">
+        <option value="">{{ field_label(form.country) }}</option>
+
+        {% for label, value in field_choices(form.country) %}
+            <option value="{{ value }}">{{ label }}</option>
+        {% endfor %}
+    </select>
+
```

Comment:
Please add a version added directive for 5.4 and target 5.4 branch, thanks

Reply:
Changed the target branch to 5.4.

However these functions were added in 5.2, so should I still mention it?

Reply:
Yes in 5.x branch we keep all versionadded info from all 5.x versions

Reply:
Done :)